lib/graphql/resolvers.ts

```diff
@@ -83,8 +83,8 @@ const resolvers = {
     createPhysician: authorize(createPhysician, [Role.Secretary]),
     upsertPhysician: authorize(upsertPhysician, [Role.Secretary]),
     createApplication: authorize(createApplication, [Role.Secretary]),
-    createRenewalApplication,
-    createReplacementApplication,
+    createRenewalApplication: authorize(createRenewalApplication, [Role.Secretary]),
+    createReplacementApplication: authorize(createReplacementApplication, [Role.Secretary]),
```

Comment:
Good catch here. `createRenewalApplication` shouldn't be wrapped in `authorize` though, because permit holders can apply for a renewal themselves